src/guides/v2.3/install-gde/prereq/apache.md

```diff
@@ -196,11 +188,94 @@ Installing and configuring Apache is basically a three-step process: install the
 1. Continue with the next section.
 
    {:.bs-callout-info}
-   Even though Apache 2.4 is provided by default with CentOS 7, you configure it like Apache 2.2. See the following section.
+   Apache 2.4 is provided by default with CentOS 7. See the following section to configure it.
+
+### Enable rewrites and .htaccess for CentOS 7
+
+1. Open `/etc/httpd/conf/httpd.conf` for editing:
+
+   ```bash
+   vim /etc/httpd/conf/httpd.conf`
+   ```
+
+1. Locate the block that starts with:
+
+   ```conf
+   <Directory /var/www/html>
+   ```
+
+1. Change the value of `AllowOverride` to `<value from Apache site>`.
```

Comment:
I think this needs clarification. This isn't a literal value they can apply here. I would rather show an example of what it should/could be and why we are editing it here (And the code sample below).

Reply:
Real configs have been added.

@dobooth , could you please review? Thank you!